node-commons/src/assembly/resources/conf/iotdb-common.properties

```diff
@@ -111,25 +114,20 @@ cluster_name=defaultCluster
 # Datatype: String
 # data_region_group_extension_policy=AUTO
 
-# The number of DataRegionGroups for each Database when using CUSTOM data_region_group_extension_policy.
-# Notice: Each Database will allocate data_region_group_per_database DataRegionGroups as soon as created.
+# When set data_region_group_extension_policy=CUSTOM,
+# this parameter is the default number of DataRegionGroups for each Database.
+# When set data_region_group_extension_policy=AUTO,
+# this parameter is the default minimal number of DataRegionGroups for each Database.
 # Datatype: Integer
-# data_region_group_per_database=1
+# default_data_region_group_per_database=2
```

Comment:
I was thinking `default_data_region_group_num_per_database` would be a better name because there is a parameter like `series_slot_num,` but it would be weird to call it `series_slot`

Reply:
Renamed